### ISSUE TITLE
v3.0 what's the meaning of the new method Language.resume_training ?

### ISSUE THREAD

**Issue by rsoika** [explosion/spaCy#6899]
I have a system running in a supervised training mode. Only the first time, when no model still exists I call the method nlp.initialize().

    if modelExists:
        nlp = spacy.load(modelPath) 
        # optimizer = nlp.resume_training()  # should I call this?
    else:
        nlp = spacy.blank(language) 
        optimizer=nlp.initialize()

     ....
     losses=nlp.update(examples, sgd=optimizer);

If I have alredy a model I start immediately with updating my model.

Now I have read about the methods `Language.resume_training` and `Language.rehearse` . Both methods are marked as '`experimental`' . From my tests so far, I can not see any changes in the results if I call these experimental methods or not. 

What is your recommendation in my scenario when I eventually have some new training data and I just want to update my existing model?

**Comment by svlandeg** [comment]
Hi! I'm going to convert your message to the discussion board, which is more suited for these kind of discussions among the community. We try to keep the issue tracker focused mainly on bugs.

You'll probably be getting an automatic email that this is closed but it really isn't - it'll just be converted and forwarded to the new thread ;-)
